### PR TITLE
HASH.dev Homepage

### PR DESCRIPTION
**Pull request by nathggns**
## 🌟 What is the purpose of this PR?

Introduces a new HASH.dev NextJS site consisting of a landing page, with the ability to subscribe to our newsletter and be tagged as a HASH.dev user. Also includes a Storybook instance. 

## ⚠️ Known issues

Having to use a alpha version of Storybook, and also pinning to an out of scope version of Prettier to fix an issue. 

## 🚫 Blocked by

N/A

## 🐾 Next steps

Introduction of a blog

## 🔍 What does this change?

Nothing – all new. 

### 📜 Does this require a change to the docs?

No

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1201927504137297/1201694916711102/f) _(internal)_

## 🛡 What tests cover this?

- None

## ❓ How to test this?

Check it matches figma design. 

## 📹 Demo

localhost_3000_ (8)

### CODE REVIEW

**Comment by nonparibus** [comment]
@nathggns Any idea why deployment has failed? @MaggieAppleton and I will wait on review until then.

**Comment by nonparibus** [comment]
@nathggns When there's a deploy link can you stick it here? I can't see one at https://vercel.com/hashintel/hashdev

**Comment by nathggns** [comment]
@nonparibus @MaggieAppleton Deployment is at https://dev-taupe.vercel.app/ – The URL is linked to this branch, and the fontawesome kit is set up to work on that URL. I haven't yet figured out how to make the fontawesome kit work with commit deployments.

File: sites/dev/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+              </Link>
+            </Stack>
+          </Typography>
+        </Stack>
+        <Stack direction="row" spacing={{ xs: 6, md: 10 }}>
+          <FooterSection label="Resources">
+            <FooterLink href="https://hash.ai/careers">Careers</FooterLink>
+            <FooterLink href="https://hash.ai/contact">Contact Us</FooterLink>
+            <FooterLink href="https://hash.ai/legal/terms">
+              Terms of Service
```
Comment (nonparibus) [inline sites/dev/src/components/Footer.tsx:128]:
```suggestion
              Terms
```

Can we shorten this to 'Terms' and also add a 'Privacy' link beneath pointing to https://hash.ai/legal/privacy

**Comment by nonparibus** [comment]
Can we change the linked word 'HASH' in the sentence "HASH is an organization building" to point to https://hash.ai/about? Given the paragraph provides context to the org itself, I feel this is the appropriate page to link out to.

**Review by nonparibus — CHANGES_REQUESTED** [review]
At some screen sizes the whitespacing beneath the HASH project GitHub button seems to screw up. Generally this is good-to-go, though. Left a couple of other comments separately re: new/changing links.
 
Screenshot 2022-04-05 at 10 38 32

**Comment by nathggns** [comment]
> At some screen sizes the whitespacing beneath the HASH project GitHub button seems to screw up. Generally this is good-to-go, though. Left a couple of other comments separately re: new/changing links.

Thanks for this. It's one of the known issues I mentioned which I thought you didn't want me to spend time fixing, but I'll take a look. 

image

File: sites/dev/src/pages/index.page.tsx [hunk 2]
```diff
@@ -0,0 +1,594 @@
...
+          <Project
+            title={<Box sx={{ mt: { xs: 0, md: 5, lg: 8 } }}>HASH</Box>}
+            buttons={
+              <Button
+                href="https://github.com/hashintel/hash/tree/main/packages/hash"
+                endIcon={
+                  <FaIcon name="arrow-up-right-from-square" type="solid" />
+                }
+              >
+                Download on Github
```
Comment (nonparibus) [inline sites/dev/src/pages/index.page.tsx:239]:
```suggestion
                Download on GitHub
```

File: sites/dev/src/pages/index.page.tsx [hunk 3]
```diff
@@ -0,0 +1,594 @@
...
+          <Project
+            title="hEngine"
+            buttons={
+              <Button
+                href="https://github.com/hashintel/hash/tree/main/packages/engine"
+                endIcon={
+                  <FaIcon name="arrow-up-right-from-square" type="solid" />
+                }
+              >
+                Download on Github
```
Comment (nonparibus) [inline sites/dev/src/pages/index.page.tsx:285]:
```suggestion
                Download on GitHub
```

**Review by nonparibus — APPROVED** [review]

**Review by nonparibus — DISMISSED** [review]

**Comment by nonparibus** [comment]
@CiaranMn When this has merged let's change the Vercel deployment to point back to the `main` branch, and ice the temporary deploy branch. Thanks again for your help in getting this up on Tue night.

**Comment by MaggieAppleton** [comment]
This all looks great to me. Nice job @nathggns ! 🎉

**Review by MaggieAppleton — APPROVED** [review]
